packages/cardpay-sdk/sdk/token-bridge-home-side.ts

```diff
@@ -47,39 +47,53 @@ const bridgedTokensQuery = `
 export default class TokenBridgeHomeSide implements ITokenBridgeHomeSide {
   constructor(private layer2Web3: Web3) {}
 
+  async relayTokens(txnHash: string): Promise<TransactionReceipt>;
   async relayTokens(
     safeAddress: string,
     tokenAddress: string,
     recipientAddress: string,
     amount: string,
     txnOptions?: TransactionOptions,
     contractOptions?: ContractOptions
+  ): Promise<TransactionReceipt>;
+  async relayTokens(
+    safeAddressOrTxnHash: string,
+    tokenAddress?: string,
+    recipientAddress?: string,
+    amount?: string,
+    txnOptions?: TransactionOptions,
+    contractOptions?: ContractOptions
   ): Promise<TransactionReceipt> {
+    let { nonce, onNonce, onTxnHash } = txnOptions ?? {};
+    if (!tokenAddress) {
```

Comment:
ditto with the comments above about testing whether the value you actually care about is shaped like an address or txnHash instead of testing a side effect